src/transformers/models/bart/modeling_flax_bart.py

```diff
@@ -1467,8 +1467,8 @@ def prepare_inputs_for_generation(
         self,
         decoder_input_ids,
         max_length,
-        attention_mask: Optional[jnp.DeviceArray] = None,
-        decoder_attention_mask: Optional[jnp.DeviceArray] = None,
+        attention_mask: Optional[jnp.Array] = None,
+        decoder_attention_mask: Optional[jnp.Array] = None,
```

Comment:
```suggestion
        attention_mask: Optional[jax.Array] = None,
        decoder_attention_mask: Optional[jax.Array] = None,
```

Should be `jax.Array` instead of `jnp.Array`.

Reply:
Oh yes, thanks. Changed.